config/releases.exs

```diff
@@ -2,18 +2,42 @@
 # from environment variables. You can also hardcode secrets,
```

Comment:
As mentioned in previous remark, we should also define the ENV variables in `config/prod.exs`.